src/app/admin/page.tsx

```diff
@@ -58,8 +58,10 @@ export type Member = {
 };
 
 export default async function AdminPage({ searchParams }: { searchParams?: { page?: string } }) {
-    const user = await currentUser();
-    if (!user?.publicMetadata.isAdmin) {
+    const session = await auth();
+    if (!session?.user) {
+        return notFound();
+    } else if (!(session?.user).isCommittee) {
```

Comment:
```suggestion
    if (!session?.user?.isCommittee) {
```

Can you change this to be only one `if`?